src/_FAQ.mdx

```diff
@@ -0,0 +1,57 @@
+import Nav from 'react-bootstrap/lib/Nav';
+import NavItem from 'react-bootstrap/lib/NavItem';
+import NavDropdown from 'react-bootstrap/lib/NavDropdown';
+import MenuItem from 'react-bootstrap/lib/MenuItem';
+import { Playground } from 'docz';
+
+# Frequently Asked Questions
+The Blue Design System Mission Statement: Establish a system that provides a single source of truth to ease the creation of high quality user experiences.
+
+## WHAT IS BLUE DESIGN SYSTEM?
```

Comment:
Let's avoid all caps in the markdown, we can always restyle this to caps with CSS if we decide to later.